apps/web/src/routes/_content/resources/podcasts/$podcastId.tsx

```diff
@@ -80,37 +94,92 @@ function Podcast() {
                 </div>
               </div>
 
-              <div className="col-span-2 my-4 flex flex-col md:mt-0">
+              <div className="w-full max-w-2xl mt-4 flex flex-col md:mt-0">
                 <div>
-                  <h2 className="mb-2 max-w-lg text-2xl font-bold text-blue-800 sm:text-4xl">
+                  <h2 className="text-2xl md:text-5xl  md:font-medium text-white mb-5 lg:mb-8">
                     {podcast?.name}
                   </h2>
 
-                  <div className="mt-2 text-sm">
-                    <h5 className="font-light italic">
-                      {podcast.host}, {podcast.lastUpdated.toString()}.
-                    </h5>
+                  <div className="flex flex-wrap mb-5 lg:mb-8">
```

Comment:
add gap to prevent this issue 
![image](https://github.com/user-attachments/assets/12f4a798-3ca2-496f-b566-2e740c8ad106)